app/src/main/java/org/schabi/newpipe/local/playlist/LocalPlaylistFragment.java

```diff
@@ -381,10 +381,11 @@ public void sharePlaylist() {
         disposables.add(playlistManager.getPlaylistStreams(playlistId)
                 .flatMapSingle(playlist -> Single.just(playlist.stream()
                         .map(PlaylistStreamEntry::getStreamEntity)
-                        .map(StreamEntity::getUrl)
-                        .collect(Collectors.joining("\n"))))
+                        .map(streamEntity -> streamEntity.getTitle() + " " + streamEntity.getUrl())
```

Comment:
I think it is better to swap title and URL, since URLs are usually all the same length, while titles are not. This way the urls would somewhat align better and be easier to click, and titles would still all start roughly from the same column. Anyway, if you disagree with me it's fine, we can keep it as it is.